src/util/time.js

```diff
@@ -86,3 +189,87 @@ export const getStreamTime = ({
 
   return callback(null, streamTime);
 };
+
+export const seekToStreamTime = ({
+  streamTime,
+  playlist,
+  retryCount = 2,
+  seekTo,
+  pauseAfterSeek = true,
+  tech,
+  callback
+}) => {
+
+  if (!callback) {
+    throw new Error('seekToStreamTime: callback must be provided');
+  }
+
+  if (typeof streamTime === 'undefined' || !playlist || !seekTo) {
+    return callback({
+      message: 'seekToStreamTime: streamTime, seekTo and playlist must be provided',
+      newTime: null
```

Comment:
Why provide `newTime`?

Reply:
just to have a consistent negative result for `newTime`. This way the caller doesn't need to check for undefined

Reply:
But the time is the second argument to the callback, the user wouldn't think there would be anything but error information in the error object.

Reply:
ah true 👍